src/main/java/uk/gov/companieshouse/api/accounts/controller/CompanyAccountController.java

```diff
@@ -15,23 +13,24 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import uk.gov.companieshouse.api.accounts.AttributeName;
-import uk.gov.companieshouse.api.accounts.CompanyAccountsApplication;
 import uk.gov.companieshouse.api.accounts.model.entity.CompanyAccountEntity;
 import uk.gov.companieshouse.api.accounts.model.rest.CompanyAccount;
 import uk.gov.companieshouse.api.accounts.service.CompanyAccountService;
 import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
 import uk.gov.companieshouse.api.accounts.transaction.Transaction;
 import uk.gov.companieshouse.api.accounts.transformer.CompanyAccountTransformer;
 import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
-import uk.gov.companieshouse.logging.Logger;
-import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.api.LogContext;
+import uk.gov.companieshouse.logging.api.LogHelper;
+import uk.gov.companieshouse.logging.api.LogType;
+import uk.gov.companieshouse.logging.api.LoggerApi;
 
 @RestController
 @RequestMapping(value = "/transactions/{transactionId}/company-accounts", produces = MediaType.APPLICATION_JSON_VALUE)
 public class CompanyAccountController {
 
-    private static final Logger LOGGER = LoggerFactory
-            .getLogger(CompanyAccountsApplication.APPLICATION_NAME_SPACE);
+    @Autowired
+    private LoggerApi accountsLogger;
```

Comment:
we should not be replacing the `LoggerFactory.getLogger` functionality with a  specific new implementation of a LoggerApi - the library changes need to be reworked for the new helper functionality not introduce a completely new logger type

Reply:
The LoggerAPi is a convinient wrapper, under the hood it uses the  LoggerFactory
            .getLogger(CompanyAccountsApplication.APPLICATION_NAME_SPACE);. Instead of getting a logger in every class, the LoggerApi bean is initialised at the loading of the context and available in every class via auto-wiring.

Reply:
The changes doesn't replace the LoggerFactory.getLogger or any part of the structured-logging functionality.

Reply:
The LoggerAPI class should have been implemented as a helper rather than a wrapper.
Using it as a wrapper has forced us to change our API here to use a different logger and would require larger rework in other API's that want to take advantage of the new functionality in the library.

The library should be changed to remove this new LoggerApi interface and expose the functionality as helpers that can be used by the existing Logger class